roles/mongo_7_0/tasks/main.yml

```diff
@@ -12,22 +12,75 @@
     - install
     - install:configuration
 
-- name: Install pip3
+- name: Update apt cache
   ansible.builtin.apt:
-    name: python3-pip
+    update_cache: true
+
+- name: Install dependencies for adding new repositories
+  ansible.builtin.apt:
+    name:
+      - software-properties-common
+      - gpg-agent
+    state: present
+
+- name: Add GPG key for deadsnakes PPA
+  ansible.builtin.apt_key:
+    url: https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xF23C5A6CF475977595C89F51BA6932366A755776
+    state: present
+
+- name: Add deadsnakes PPA for Python 3.10
+  ansible.builtin.apt_repository:
+    repo: ppa:deadsnakes/ppa
+    state: present
+
+- name: Install venv
+  ansible.builtin.apt:
+    name:
+      - python3.10
+      - python3.10-venv
```

Comment:
Can't we use the same python version of the host?

Reply:
We're getting the following error when use Python3.12 with [mongodb_user](https://docs.ansible.com/ansible/latest/collections/community/mongodb/mongodb_user_module.html) for somewhat reason.

`Unable to connect to database: 'RawConfigParser' object has no attribute 'readfp'"`

It appears to be compatibility issue but the module is updated, the venv where we run the playbook too...